### PR TITLE
Net connection handler

### PR DESCRIPTION
**Pull request by Dekim12**

### CODE REVIEW

File: src/screens/WordsScreen/WordsScreen.js [hunk 1]
```diff
@@ -2,11 +2,30 @@
...
+import {
+  MOVEMENT_FUNC_NAMES,
+  NET_CONNECTION_SCREEN,
+  NET_CONNECTION_ID,
+} from '../../constants'
 import styles from './style'
 
+type OverlayObjType = {
+  component: {
+    name: String,
```
Comment (LenaZhuk) [inline src/screens/WordsScreen/WordsScreen.js]: `string`?

File: src/screens/WordsScreen/WordsScreen.js [hunk 2]
```diff
@@ -2,11 +2,30 @@
...
+    id: string
+  }
+}
+
+type NetStateType = {
```
Comment (LenaZhuk) [inline src/screens/WordsScreen/WordsScreen.js]: You could import this from netinfo module

File: src/screens/WordsScreen/WordsScreen.js (hunk 1 above)
Comment (Dekim12) [reply src/screens/WordsScreen/WordsScreen.js]: oyy... yes

File: src/screens/WordsScreen/WordsScreen.js [hunk 3]
```diff
@@ -18,10 +37,33 @@ type State = {
...
+
   constructor(props: Props) {
     super(props)
 
     this.state = { permissionVisible: false, permissionResolve: null, }
+    this.connectionIndicator = true
+  }
+
+  connectionHandler = (netState: NetStateType): void => {
+    if (!netState.isConnected) {
```
Comment (LenaZhuk) [inline src/screens/WordsScreen/WordsScreen.js]: Destructuring?

**Pull request closed without merging**